testsuite/tests/lazy/lazy3.ml

```diff
@@ -2,8 +2,10 @@
    ocamlopt_flags += " -O3 "
 *)
 
+let get_domain_id () = Domain.self () |> Domain.string_of_id |> int_of_string
```

Comment:
It looks like this test does not use the domain id. Let's remove this definition and its uses.